scivision.yml

```
@@ -0,0 +1,25 @@
+sources:
+  plankton_single:
+      description: Load a single labeled images from CEFAS zooplankton dataset
+      origin: 
+      driver: intake_xarray.image.ImageSource
+      parameters:
+        id:
+          description: which filename
+          type: str
+          default: Pia1.2017-10-03.1726+N00296780_hc
+      args:
+        urlpath: './data/images/{{id}}.tif'
+        storage_options: {'anon': True}
+        exif_tags: True
+
+  plankton_multiple:
+      description: Labeled images from CEFAS zooplankton dataset
+      origin: 
+      driver: intake_xarray.image.ImageSource
+      args:
+        urlpath: './data/images/{filename}.tif'
+        chunks: {}
+        storage_options: {'anon': True}
+        coerce_shape: [256, 256]
```

Coerce to the maximum image size, so we don't crop any image
```suggestion
        coerce_shape: [832, 1040]
```

---

There isn't really a better way than hard-coding the size for now. We can leave it to the participants to resize/rescale the images if they want to.